### ISSUE TITLE
UI/Help package tokens do not work for VM installations (v10+)

### ISSUE THREAD

**Issue by m-s-** [criticalmanufacturing/cli#313]
CLI will place environment specific tokens in config.json, index.html and ngsw.json that should be replaced by DF when installing the package. 
However, the tokens supported by the container entrypoints and the traditional DF installation are not compatible, causing the package to install with tagging in VMs and the UIs to not work anymore.

**Comment by m-s-** [comment]
A work around would be to edit the generated package and replacing the tokens with ones supported by the traditional installation process:

### config.json
```
$(TENANT_NAME) -> $(Product.Tenant.Name)
$(APPLICATION_PUBLIC_HTTP_TLS_ENABLED) -> $(Product.ApplicationServer.EnableSsl)
$(APPLICATION_PUBLIC_HTTP_ADDRESS) -> $(Product.ApplicationServer.Address)
$(APPLICATION_PUBLIC_HTTP_PORT) -> $(Product.ApplicationServer.Port)
$(SYSTEM_NAME) -> $(Product.SystemName)
"$(SECURITY_PORTAL_STRATEGY_LOCAL_AD_DEFAULT_DOMAIN) -> $(Product.Security.Domain)
```
### index.html and ngsw.json
replace `$(APPLICATION_BASE_HREF)` with `/`

**Comment by m-s-** [comment]
This is technically a Deployment Framework bug, not a CLI bug, but I've tagged the issue as such anyway for easier reference.

**Comment by socujun** [comment]
Thanks for your guidance, my page can also be displayed, otherwise I will be stuck in the project

**Comment by m-s-** [comment]
This is fixed by MES 10.1, in which VM DF understands the container tokens
